src/app/greetings/.greetingReducer.js

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+const BASE_URL = 'http://localhost:3000/greetings';
+
+export const getGreetingAsync = createAsyncThunk(
+    'fetchGreeting', async () => {
+        const response = await fetch(BASE_URL);
+        const { greeting } = await response.json();
+        return greeting;
```

Comment:
- [ ] What you should destructure here is `message`, not `greeting` since that is what the API returns.